### PR TITLE
Chore: Directory based components and widgets

### PR DESCRIPTION
**Pull request by C-Valen**
# Chore - Directory based components and widgets

## Checklist:

- [x] Commits consolidated
- [N/A] Changelog updated
- [x] Test cases added/updated
- [N/A] Jira ticket updated

## Summary:

As discussed in internal team calls this PR converts `@/components` and `@/widgets` into to directory based vue components.
This allows to break down large components in smaller parts providing more flexibility and an easier maintainability, aiming to adopt an atomic component design pattern.

## Changes:

- Use directories for components under `@/components` and `@/widgets`
- Relocate `@/widgets` to `/src`
- Update imports
- Update snapshots (cause of components data id)

## Considerations:

- This PR only provides the initial structure for directory based components, it will be required to refactor and adapt individual components that can benefit from this new format.
- `widgets` directory made sense to me to be relocated at `./src` to be at same level as `views` and `components`

### CODE REVIEW

**Comment by C-Valen** [comment]
Update: Use names component index templates.

File: .eslintrc.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "vue/multi-word-component-names": [
       "error",
       {
-        "ignores": ["Modal", "Tabs", "Toast", "Login", "Navbar", "Settings"]
+        "ignores": ["Modal", "Tabs", "Toast", "Login", "Navbar", "Settings", "index"]
```
Comment (MrMarble) [inline .eslintrc.json:79]: Is this change needed?

Comment (C-Valen) [reply .eslintrc.json:79]: Not really, it was for previous approach but is not required now.

**Review by superbuggy — APPROVED** [review]

**Review by MrMarble — APPROVED** [review]
